src/adv.py

```diff
@@ -54,16 +54,21 @@
 adventurer = Player("Adventurer", room['outside'])
 print(f"Welcome to the Cave press q to quit")
 
-
 while True:
 
     print(f"{adventurer.current_room}")
```

Comment:
Very compact logic. Well done! 

You can make it more readable with comments and more OOP practices as you already mentioned